packages/mongo-nav/src/helpers/Icons.tsx

```diff
@@ -1,20 +1,149 @@
 import React from 'react';
-import { css } from '@leafygreen-ui/emotion';
+import { css, cx } from '@leafygreen-ui/emotion';
 import { HTMLElementProps } from '@leafygreen-ui/lib';
 
 type ProductIconProps = HTMLElementProps<'svg'> & {
-  active: boolean;
+  active?: boolean;
+  variant?: 'small' | 'default';
 };
 
-export function AtlasIcon({ active, ...rest }: ProductIconProps) {
+export function CloudManagerIcon({
+  variant = 'default',
+  active = false,
+  className,
+  ...rest
+}: ProductIconProps) {
+  const size = variant === 'small' ? 10 : 18;
```

Comment:
For future flexibility, we may want to render the small version based on the size, not the other way around.